### ISSUE TITLE
model does not converge

### ISSUE THREAD

**Issue by echolijinghui** [yulunzhang/RDN#1]
hi，sorry to bother you,but I have a problem that model does not converge,what should I do?can you give me some suggestions?thank you so much.the loss function i used is l1 loss 

![default](https://user-images.githubusercontent.com/37646369/39221938-6570b666-486c-11e8-8518-bbf5170f9c2f.PNG)

**Comment by yulunzhang** [comment]
Hi,

Did you run the provided torch code? What training script did you use?

If we train the network from scratch (e.g., for BI, X2), the loss should converge fast at the beginning.
![bix2_loss](https://user-images.githubusercontent.com/22829440/39261611-03bd5482-488b-11e8-97b1-725fce303eb2.PNG)

If we fine-tune the network based on previous trained models, the loss might fluctuate at the beginning. After the learning rate decreases, the loss would also trend to decrease. 
![bix2_finetune_loss](https://user-images.githubusercontent.com/22829440/39261703-4a699f44-488b-11e8-8c12-32c24b6f49d6.PNG)

However, no matter we train from scratch or fine-tune, the L1 loss value should not be so large.  So, could you give more training details about your posted results? Thanks.
